conda-recipes/scvi/meta.yaml

```diff
@@ -0,0 +1,82 @@
+{% set name = "scvi" %}
+{% set version = "0.1.1" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "ca87155662d85605f86c5e86b7b9f64d881b882177b9642fff8f0ea215c6cb1a" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  host:
+    - python
+    - setuptools
+    - pip >=9.0.1
+    - flake8 ==3.5.0
+    - coverage ==4.5.1
+    - sphinx ==1.7.1
+    - pytest ==3.4.2
+    - pytest-runner ==2.11.1
+    - numpy ==1.14.0
+    - torch ==0.4.0
+    - matplotlib ==2.2.2
+    - scikit-learn ==0.19.1
+    - scipy ==1.0.0
+    - h5py
+    - codecov
+    - pandas ==0.23.0
+    - loompy ==2.0.9
+  run:
+    - python
+    - pip >=9.0.1
+    - flake8 ==3.5.0
+    - coverage ==4.5.1
+    - sphinx ==1.7.1
+    - pytest ==3.4.2
+    - pytest-runner ==2.11.1
+    - numpy ==1.14.0
+    - torch ==0.4
+    - matplotlib ==2.2.2
+    - scikit-learn ==0.19.1
+    - scipy ==1.0.0
+    - h5py
+    - codecov
+    - pandas ==0.23.0
+    - loompy ==2.0.9
+
+test:
+  imports:
+    - scvi
+    - scvi.dataset
+    - scvi.metrics
+    - scvi.models
+  requires:
+    - pytest
+
+about:
+  home: https://github.com/YosefLab/scVI
+  license: MIT
+  license_family: MIT
+  license_file: 'LICENSE'
+  summary: Single-cell Variational Inference
+  description: Single-cell Variational Inference
+  doc_url: https://scvi.readthedocs.io
+  dev_url: https://github.com/YosefLab/scVI
+
```

Comment:
It'd be better to put the conda stuff in a separate pull request than the remote loom stuff.

Reply:
(That keeps makes it easier to revert one change without the other, and it's more readable for anyone looking through the git history to see how repository has evolved.)

Reply:
FYI, you might use `git cherry-pick` to move the conda commit to another branch. See also `git rebase -i` and `git push origin yl/remote_loom`.